spec/yaml_extend_spec.rb

```diff
@@ -8,6 +8,23 @@
 
 RSpec.describe YAML,'#ext_load_file' do
   context 'Test inheritance feature' do
+
+    it 'multi-file inheritance precedence order' do
```

Comment:
Probably could be expanded to something like 'follows the correct inheritance precedence order when extending from multiple files'